net-dns/dnsdist/dnsdist-1.1.0-r1.ebuild

```diff
@@ -1,84 +0,0 @@
-# Copyright 1999-2017 Gentoo Foundation
```

Comment:
I'm pretty sure this commit leaves us with no unmasked version, and broken Manifest. I suppose you want to reorder the commits.

Also, please indicate what the bugs are about instead of their numbers. Numbers have little usefulness as people end up having to look both bugs up to know what's going on. Put the bug URLs in the footer instead, like:

```
Bug: https://bugs.gentoo.org/628534
Bug: https://bugs.gentoo.org/628578
```

The hooks will read that and leave a comment on both bugs.